WordPress/Classes/ViewRelated/Reader/ReaderDetailWebviewViewController.swift

```diff
@@ -208,6 +226,46 @@ class ReaderDetailWebviewViewController: UIViewController, ReaderDetailView {
         coordinator?.share(fromView: sender)
     }
 
+    /// Start following the scroll view to hide nav bar and toolbar
+    /// Only if VoiceOver is not active
+    ///
+    private func followScrollView() {
+        if isFollowingScrollView,
+            UIAccessibility.isVoiceOverRunning {
+            return
+        }
+
+        if let navigationController = navigationController as? ScrollingNavigationController {
+            navigationController.followScrollView(scrollView, delay: 50.0, followers: [NavigationBarFollower(view: toolbarContainerView, direction: .scrollDown)])
+            navigationController.shouldUpdateContentInset = false
+            isFollowingScrollView = true
+        }
+    }
+
+    /// Stop following the scroll view to hide nav bar and toolbar
+    ///
+    private func stopFollowingScrollView() {
+        if let navigationController = navigationController as? ScrollingNavigationController {
+            navigationController.stopFollowingScrollView(showingNavbar: true)
+            isFollowingScrollView = false
+        }
+    }
+
+    /// Show the nav bar when scrolling to top
+    ///
+    func scrollViewShouldScrollToTop(_ scrollView: UIScrollView) -> Bool {
```

Comment:
Should this method be grouped in the UIScrollViewDelegate extension below?

Reply:
Done in bb12fd1b9e109fa7ce742832400510367ef8e00e